sidechain/test/src/sidechain_block_builder.rs

```diff
@@ -35,114 +35,60 @@ type Seed = [u8; 32];
 const ENCLAVE_SEED: Seed = *b"12345678901234567890123456789012";
 
 pub struct SidechainBlockBuilder {
-	signer: ed25519::Pair,
-	number: BlockNumber,
-	parent_hash: BlockHash,
-	parentchain_block_hash: H256,
-	signed_top_hashes: Vec<H256>,
-	encrypted_payload: Vec<u8>,
-	shard: ShardIdentifier,
-	timestamp: Timestamp,
+	signer: Option<ed25519::Pair>,
```

Comment:
This seems to be always `Some(_)`, never `None`? In that case we don't need the `Option<>`.